source/sass/buyers-guide/includes/draft-product.scss

```diff
@@ -1,7 +1,7 @@
 // product draft rules
 .draft-product::before {
   content: "This is a draft.";
-  color: #ffa7a7;
+  color: $light-red;
```

Comment:
Colour nit: unlike green or blue, there is no "light" red. It goes from red through a few named colours to pink, so this would be something either "salmon" or "rose" =)

Reply:
I like current our color naming system 'cuz everything looks intuitive to me. However I'm not an expert in this area so if you'd like to propose a new naming convetion we should create a ticket and get designers to chime in. 🎨

Reply:
the scheme makes sense, it just leads to some color names that out of context don't make a lot of sense =)